pkgs/development/python-modules/nudatus/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, python3Packages
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "nudatus";
+  version = "0.0.5";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-uoSo4CLLSvD5lRdjg/X/n6g9V97TztxqpNsWh/iMrOw=";
+  };
+
+  nativeBuildInputs = with python3Packages; [
+    setuptools
+    wheel
+  ];
+
```

Comment:
We should execute the tests https://github.com/ZanderBrown/nudatus/tree/main/tests